CFC_WebApp/clients/socialmedia/settings.json.sample

```diff
@@ -0,0 +1,4 @@
+{
+    "fb_app_secret":"ask sid",
```

Comment:
json.sample doesn't seem to be similar to json. I presume you want to delete .json, but then if somebody takes .json.sample and copies it over, I don't think that it will work.